tradesman/model.py

```diff
@@ -142,12 +142,12 @@ def import_buildings(self):
 
         building_import(self.__model_place, self._project, self.__osm_data)
 
-    def build_population_synthesizer_data(self):
+    def build_population_synthesizer_data(self, threads: None, sample=0.02):
```

Comment:
We need docstrings for this one. And a warning that the sample needs to be low.  I would also reduce the default to 1%